.changeset/fair-carrots-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+"@snapwp/next": patch
+---
+
+Bug-Fix: remote patterns in users config being ignored
```

Comment:
Since these get converted to the change log, they should also use [Conventional Commits](https://gist.github.com/qoomon/5dfcdf8eec66a051ecd85625518cfd13)

```suggestion
fix: ensure user-defined `remotePatterns` aren't overridden by `withSnapWP()`
```